modules/c/presto/presto.cpp

```diff
@@ -54,18 +54,32 @@ typedef struct _ModPicoGraphics_obj_t {
     DisplayDriver *display;
 } ModPicoGraphics_obj_t;
 
-
+// There can only be one presto display, so have a global pointer
+// so that core1 can access it.  Note it also needs to be in the
+// Micropython object to prevent GC freeing it.
+static ST7701 *presto;
```

Comment:
This is slightly sad but seemed the simplest approach.

Reply:
I think we still have some of this pattern knocking about, if it works it works!